src/views/dashboard/Calendar.vue

```diff
@@ -65,18 +65,30 @@ export default {
   },
   methods: {
     eventClick (info) {
-      if (info.event.url) {
-        return
-      }
-
       info.jsEvent.preventDefault() // don't let the browser navigate
-      this.$buefy.dialog.alert({
-        title: info.event.title,
-        message: '<strong>Starts: </strong>' + moment(info.event.start).format('D MMMM Y H:mm')
-          + '</br><strong>Ends: </strong>' + moment(info.event.end).format('D MMMM Y H:mm')
-          + '<hr>' + info.event.extendedProps.description,
-        canCancel: '[escape, outside]'
-      })
+
+      if (info.event.url !== '') { // If the event has a non-empty link field
+        this.$buefy.dialog.confirm({
+          title: info.event.title,
```

Comment:
these 2 props are the same for both if/else, I'd rather move it outside of if/else to reduce copypaste

Reply:
The onConfirm is different. We want that to only happen if there is a link for the event. I agree that it could be nicer but Diederik could not really figure it out. Do you maybe have an idea?

Reply:
And the canCancel has a button for the case where there is a link